### DISCUSSION TITLE
How to represent dual-personality ports on switches.

### DISCUSSION THREAD

**Discussion by Madrov** [netbox-community/devicetype-library#1278] [General]
Hello,

I wonder what's the best way to represent a port that has two function. For example, the HPE ProCurve 2510 series have interfaces described as:

"4 dual-personality port. Each port can be used as either an RJ-45 10/100/1000 port or an open mini-GBIC slot (for use with mini-GBIC transceivers)" (Ref. https://support.hpe.com/hpesc/public/docDisplay?docId=emr_na-c02620067)

Thank you!

**Comment by danner26** [comment]
The best way to model it is by the devices default. If out of the box the interface is a 1000base-t port, then that is what it should be modelled as
